### PR TITLE
SREP1775 updated dedicated-admins group management

### PR DESCRIPTION
**Pull request by sheriff-rh**
@jewzaam  @mwoodson please take a look

### CODE REVIEW

File: modules/dedicated-managing-dedicated-administrators.adoc [hunk 1]
```diff
@@ -6,24 +6,11 @@
 =  Managing {product-title} administrators
 
 Administrator roles are managed using a `dedicated-admins` group on the cluster.
-Existing members of this group can edit membership. To view a list of current
-dedicated administrators by user name, you can use the following command:
+Existing members of this group can edit membership via the OpenShift Cluster
```
Comment (jewzaam) [inline modules/dedicated-managing-dedicated-administrators.adoc]: Recommend linking to OCM:  https://cloud.redhat.com/openshift

File: modules/dedicated-managing-dedicated-administrators.adoc [hunk 2]
```diff
@@ -6,24 +6,11 @@
...
-$ oc adm groups add-users dedicated-admins <user_name>
-----
-
-To remove an existing user from the `dedicated-admins` group:
-
-----
-$ oc adm groups remove-users dedicated-admins <user_name>
-----
+. In *Cluster Details*, click *Users* -> *Add User*.
+. Enter the *Username* and *Group* (dedicated-admins), then click the *Add* button.
```
Comment (jewzaam) [inline modules/dedicated-managing-dedicated-administrators.adoc]: - only click "Add User" for the first user
- missing how to remove a user

**Review by jewzaam — CHANGES_REQUESTED** [review]

File: modules/dedicated-managing-dedicated-administrators.adoc (hunk 1 above)
Comment (sheriff-rh) [reply modules/dedicated-managing-dedicated-administrators.adoc]: Agreed - added a link.

File: modules/dedicated-managing-dedicated-administrators.adoc (hunk 2 above)
Comment (sheriff-rh) [reply modules/dedicated-managing-dedicated-administrators.adoc]: I removed the step *Add User*. Can you provide me steps to remove a user?

Comment (jewzaam) [reply modules/dedicated-managing-dedicated-administrators.adoc]: In the "Users" tab click the X to the right of the user + group you'd like to remove.

Screenshot from 2019-08-13 18-34-30

Comment (jewzaam) [reply modules/dedicated-managing-dedicated-administrators.adoc]: When adding the first user you have to click the "Add User" button.  Here's a screenshot.

Screenshot from 2019-08-20 16-21-43

**Review by jewzaam — CHANGES_REQUESTED** [review]

File: modules/dedicated-managing-dedicated-administrators.adoc (hunk 2 above)
Comment (sheriff-rh) [reply modules/dedicated-managing-dedicated-administrators.adoc]: Updated the steps again, thanks.

**Review by jewzaam — CHANGES_REQUESTED** [review]

**Comment by sheriff-rh** [comment]
Changes commited. Okay to merge @jewzaam ?

**Pull request merged**